tests/test-get-products.js

```diff
@@ -90,23 +90,4 @@ describe('fxpay.getProducts()', function() {
 
     helper.server.respond();
   });
-
-  it('should pass through init errors', function (done) {
-    // Trigger an init error:
-    helper.appSelf.error = {name: 'INVALID_MANIFEST'};
```

Comment:
actually, this was interesting because ``products.all()`` doesn't need ``appSelf`` at all so it doesn't need to pass through any initialization errors. That made me realize lazy access to appSelf solves a lot of problems. The concept of initialization always bothered me and I think you also had some early review comments about how up front initialization was clunky :)